cclient/request/get_balances.h

```diff
@@ -22,21 +22,22 @@ typedef struct {
   /**
    * List of addresses you want to get the confirmed balance for.
    */
-  flex_hash_array_t* addresses;
+  hash243_queue_t addresses;
   /**
    * List of hashes, if present calculate the balance of addresses
    * from the PoV of these transactions, can be used to chain bundles.
    */
-  flex_hash_array_t* tips;
+  hash243_queue_t tips;
 } get_balances_req_t;
 
 get_balances_req_t* get_balances_req_new();
 void get_balances_req_free(get_balances_req_t** req);
-get_balances_req_t* get_balances_req_add_address(get_balances_req_t* req,
-                                                 char* addr);
-get_balances_req_t* get_balances_req_add_tips(get_balances_req_t* req,
-                                              char* tip);
-
+/*
+retcode_t get_balances_req_add_address(get_balances_req_t* req,
+                                       flex_trit_t const* const addr);
+retcode_t get_balances_req_add_tips(get_balances_req_t* req,
+                                    flex_trit_t const* const tip);
+*/
```

Comment:
IMHO we should definitely continue to encapsulate these, even if we use the new containers.
Changing the inner container for some reason one day would be much easier.

Reply:
better have one container model for keeping simple in cclient.